internal/gps/internal/paths/paths_test.go

```diff
@@ -2,23 +2,24 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package internal
+package paths
 
 import "testing"
 
-func TestIsStdLib(t *testing.T) {
+func TestIsStandardImportPath(t *testing.T) {
 	fix := []struct {
 		ip string
 		is bool
 	}{
 		{"appengine", true},
 		{"net/http", true},
 		{"github.com/anything", false},
+		{"github.com", false},
 		{"foo", true},
```

Comment:
This is important, but also a huge can of worms - if we want to talk about changing the implementation/expectations of this function, let's do it in a separate issue to avoid derailing this PR.